docker-library/apache-php-mysql/php.ini

```diff
@@ -4,4 +4,7 @@ error_log=/var/log/httpd/php-error.log
 display_errors=Off
 display_startup_errors=Off
 date.timezone=UTC
-
+; increase upload size for PHP
+post_max_size=20M
+upload_max_filesize=20M
+memory_limit=128M
```

Comment:
128M is default max memory value memory_limit. 8M before PHP5.2.0, 16M in PHP 5.2.0